Annotato/Annotato/WebSocket/WebSocketManager.swift

```diff
@@ -3,6 +3,7 @@ import AnnotatoSharedLibrary
 
 class WebSocketManager {
```

Comment:
Can we put the last online time related things in a extension similar to AnnotatoAuth

Reply:
Ok done